### ISSUE TITLE
jslintworks not consistent

### ISSUE THREAD

**Issue by nimo23** [angelozerr/typescript.java#227]
I have two jsx files which equal import statements at the beginning (both are within ts-directory)

```
import React, { Component } from 'react'
import ReactDOM from "react-dom";
```

one file does not complain, but the other file complains with "..has no export module". Normally both must be complained..

this is enabled:

```
"plugins": [
			{
				"name": "tslint-language-service"
			}
		],
```

**Comment by nimo23** [comment]
When deleting plugins with "tslint-language-service" from tsconfig, I expect, that typescript.java does not make any checks within my ts-files. However, it does. How can I disable this?

**Comment by angelozerr** [comment]
> "..has no export module".

Are you sure it's tslint error?

**Comment by nimo23** [comment]
I have set up eclipse javascript to use "module". I am using eclipse neon with jboss tools..maybe the error comes from jboss validator..

**Comment by nimo23** [comment]
I deleted error logs. it s okay now.
